README.md

```diff
@@ -7,15 +7,17 @@ Collection of generative AI prototypes, mainly using LLMs.
     - [Explain like I am a 3 year old](#explain-like-i-am-a-3-year-old)
     - [Suggesting personalised early-years activities](#suggesting-personalised-early-years-activities)
     - [Suggesting personalised early-years activities w/ external knowledge base](#suggesting-personalised-early-years-activities-w-external-knowledge-base)
-      - [Categorise "Tiny Happy People" activities to the EYFS areas of learning and build a vector index](#categorise-the-bbc-activities-to-the-eyfs-areas-of-learning-and-build-a-vector-index)
-      - [Generate activities based on the user's query and the "Tiny Happy People" activities](#generate-activities-based-on-the-users-query-and-the-bbc-activities)
+      - [Categorise the BBC activities to the EYFS areas of learning and build a vector index](#categorise-the-bbc-activities-to-the-eyfs-areas-of-learning-and-build-a-vector-index)
+      - [Generate activities based on the user's query and the BBC activities](#generate-activities-based-on-the-users-query-and-the-bbc-activities)
+    - [WhatsApp interface to early-years prompts](#whatsapp-interface-to-early-years-prompts)
```

Comment:
Hey, I think you might need to merge `dev` into `27`, as I suggested the adjustment of writing "Tiny Happy People" instead of "BBC activities" here in the previous PR